aspnetcore/fundamentals/startup.md

```diff
@@ -47,18 +47,58 @@ To learn more about `WebHostBuilder`, see the [Hosting](xref:fundamentals/hostin
 
 The [ConfigureServices](/dotnet/api/microsoft.aspnetcore.hosting.startupbase.configureservices) method is:
 
-* Optional.
+* Optional
 * Called by the web host before the `Configure` method to configure the app's services.
 * Where [configuration options](xref:fundamentals/configuration/index) are set by convention.
 
 Adding services to the service container makes them available within the app and in the `Configure` method. The services are resolved via [dependency injection](xref:fundamentals/dependency-injection) or from [IApplicationBuilder.ApplicationServices](/dotnet/api/microsoft.aspnetcore.builder.iapplicationbuilder.applicationservices).
 
-The web host may configure some services before `Startup` methods are called. Details are available in the [Hosting](xref:fundamentals/hosting) topic. 
+The web host may configure some services before `Startup` methods are called. Details are available in the [Hosting](xref:fundamentals/hosting) topic.
 
 For features that require substantial setup, there are `Add[Service]` extension methods on [IServiceCollection](/dotnet/api/Microsoft.Extensions.DependencyInjection.IServiceCollection). A typical web app registers services for Entity Framework, Identity, and MVC:
 
 [!code-csharp[](../common/samples/WebApplication1/Startup.cs?highlight=4,7,11&start=40&end=55)]
 
+::: moniker range=">= aspnetcore-2.1" 
+
+<a name="setcompatibilityversion"></a>
+### SetCompatibilityVersion for ASP.NET Core MVC runtime
+
+The `SetCompatibilityVersion` method allows an app to opt-in or opt-out of potentially breaking changes introduced in the ASP.NET MVC Core 2.1 and later runtime. These potentially breaking changes are generally in how the MVC subsystem behaves and how **your code** is called by the runtime. By opting in, you get the latest behavior, and the long-term behavior of ASP.NET Core runtime.
+
+The following code sets the compatibility mode to ASP.NET Core 2.1:
+
+[!code-csharp[Main](startup/sampleCompatibility/Startup.cs?name=snippet1)]
+
+We recommend you test your application using the latest version (`CompatibilityVersion.Version_2_1`). We anticipate that most applications will not have breaking changes using the latest version. 
+
+Applications that call `SetCompatibilityVersion(CompatibilityVersion.Version_2_0)` are protected from potentially breaking changes introduced in the ASP.NET Core 2.1 MVC runtime and later 2.x versions. This protection:
```

Comment:
> potentially breaking changes introduced 

Sorry, I know I'm being picky 😆 

I want to be really specific and use the term `breaking behavior changes` consistently. 

We try to avoid any breaking change in general, and we **never** do breaking API changes in a minor release.